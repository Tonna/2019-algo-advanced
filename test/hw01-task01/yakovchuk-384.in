25
1 1 2 1 2 1 3 3 1 8 3 6 4 1 2 12 14 5 9 18 18 20 13 12
8
16 7
14 10
20 7
9 5
7 5
24 3
16 3
15 8
