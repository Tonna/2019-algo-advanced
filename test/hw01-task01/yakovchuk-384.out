11
23
20
8
12
14
5
26
